tools/source/key/main.d

```diff
@@ -75,7 +78,9 @@ int main (string[] args)
             scope ret = kp.address.verify(res);
             assert(ret == msg);
         }
-        writeln("Signed message: ", res);
+        write("Signed message: ");
+        res.each!((b) => write(LHexDigits[b >> 4], LHexDigits[(b & 0b0000_1111)]));
```

Comment:
is this simpler than using some `toHex/toHexString` utility in Phobos? I'm sure there's a few.

Reply:
Erg ofc